context/app/static/js/index.jsx

```diff
@@ -13,7 +14,7 @@ import { setJsonLD } from './schema.org';
 
 ReactDOM.render(
   // eslint-disable-next-line no-undef
-  <App flaskData={flaskData} />,
+  window.location.pathname.startsWith('/iframe/') ? <Iframe flaskData={flaskData} /> : <App flaskData={flaskData} />,
```

Comment:
Adding the `iframe` like this would add its underlying components back to the main bundle undoing some of the deliberate splitting we do later on.